code/game/objects/items/holosign_creator.dm

```diff
@@ -82,6 +82,12 @@
 			qdel(H)
 		balloon_alert(user, "holograms cleared")
 
+/obj/item/holosign_creator/emp_act(severity) // Monke, emping a holosign creator makes it destroy its signs.
+	. = ..()
+	if(LAZYLEN(signs))
```

Comment:
early return

Reply:
early return? What do you mean? (I just copied the sign deletion from the attack_self proc.)